web_crawler/ksrf.py

```diff
@@ -18,7 +19,7 @@
 from selenium import webdriver
 # License: Apache License 2.0
 
-from web_crawler import DataSource, DataSourceType, DataType
+from web_crawler.web_crawler import DataSource, DataSourceType, DataType
```

Comment:
Используй условные импорты, пожалуйста.
Предложение по изменению (новая функция появилась на гитхабе сегдня):
```suggestion
if __package__:
    from web_crawler.web_crawler import DataSource, DataSourceType, DataType
else:
    from web_crawler import DataSource, DataSourceType, DataType
```
```